docs/en/finance/core-concepts/debtor-groups.md

```diff
@@ -0,0 +1,39 @@
+# Debtor Groups
+
+All debtors in BHIMA are organized into Debtor Groups.  Debtor Groups determine the accounts and billing structure for individual debtors.
+
+## Required Information
+
+Because debtor groups are principally a financial concept, we require certain financial information to create one.  These include:
+
+1. **Name** - the name of the group that will show up in labels and dropdown menus throughout the application.
+2. **Locked** - locks the group to prevent patient assignment and further invoicing of group members.
+3. **Account** - the account used in transactions involving a member of the debtor group.
+4. **Price List** - a price list to apply to the debtor group.
+5. **Max Credit** - _not implemented yet_ [#5068](https://github.com/IMA-WorldHealth/bhima/issues/5068).
```

Comment:
Even though "Max Credit" is not implemented, you should still define it if you list it.  Just leave the "not implemented" at the end.